crates/aptos/src/move_tool/mod.rs

```diff
@@ -2179,6 +2180,11 @@ pub struct Replay {
     /// If present, skip the comparison against the expected transaction output.
     #[clap(long)]
     pub(crate) skip_comparison: bool,
+
+    /// Key to use for ratelimiting purposes with the node API. This value will be used
+    /// as `Authorization: Bearer <key>`
+    #[clap(long)]
+    pub(crate) node_api_key: Option<String>,
```

Comment:
Should we just be using `RestOptions` like all the other commands, or does that not make sense here?

Reply:
`RestOptions` holds a `url` options. The `replay` command actually uses a `network` flag and then maps it to a `url`. Using both options can be confusing and changing it will just introduce an unnecessary breaking change

Reply:
Gotcha, this is fine then.

Reply:
oh, that's really annoying we should probably fix the replay to use common inputs, but I think that's out of scope here.